deploy/arax/scripts/download_database.sh

```diff
@@ -10,7 +10,7 @@
 # sftp -i /etc/ssh-key/ssh-key-SECRET_FILE -o StrictHostKeyChecking=no -pr team-expander-ops@sftp.transltr.io:/transltr-063208468694-sftp/team-expander/databases/ /
 
 # variables
-config_file="/configs/configv2.json"
+config_file="/configs/configv_secrets.json"
```

Comment:
@pahmadi8740 - I think you have a typo here? should be `/config_secrets.json` instead of `/configv_secrets.json`?